src/views/static/people/people-view.js

```diff
@@ -0,0 +1,154 @@
+import { customElement, html, LitElement } from 'lit-element';
+
+import styles from './people-view.styles';
+
+@customElement('people-view')
+export class PeopleView extends LitElement {
+  static get styles() {
+    return [styles];
+  }
+
+  hiddenElement() {
```

Comment:
this code is needlessly duplicated. This function should only be written once.

If you have a few pages that look almost the same, consider creating a component to hold all static pages and the common logic. Then you can control the content of that page with conditional rendering (`if/else` render()`).

Reply:
duplicated code is deleted in all static pages